src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java

```diff
@@ -27,8 +29,25 @@ private Node loadRootNode() {
         try {
             return Node.root(YmlUtil.load(path));
         } catch (IOException exception) {
-            log.warn("file '.cong.yml' not found, using defaults");
+            log.warn("file '" + CONF_YAML + "' or '" + CONF_YML + "' not found, using defaults");
             return Node.emptyRoot();
         }
     }
+
+    private Path getConfigPath(String fixturesRoot) {
```

Comment:
`getConfigPath` method, I assume, should not produce any side effects like throwing exceptions.
It should only return path to config file and nothing more:
```java
Optional<Path> getConfigPath(String fixturesRoot) {
Stream
  .of(".conf.yml", ".conf.yaml")
  .map( file -> path.resolve(file))
  .filter(path -> path.toFile().exists())
  .findFirst();
}
```

Reply:
The caller(method `load()`) of `getConfigPath`, then, is responsible to handle two situations:
1. The `Optional` is empty means no config file is provided
2. A config file has a "twin"(`.conf.yaml` and `.conf.yml` exist together) together.
I assume we need a static utility method somewhere to detect whether a yaml file has a "twin" - this method should consume a `Path`(full path) and to return boolean if there is a "twin"